docs/release-notes/highlights-6.3.0.asciidoc

```diff
@@ -1,9 +1,140 @@
 [[release-highlights-6.3.0]]
-== 6.3.0 release highlights
+== 6.3.0 Release Highlights
 ++++
 <titleabbrev>6.3.0</titleabbrev>
 ++++
 
 coming[6.3.0]
 
-See also <<breaking-changes>> and <<release-notes-6.3.0>>.
\ No newline at end of file
+Each release of {kib} brings new features and product improvements.
+Here are the highlights of the features that were added in 6.3 and
+how the user experience improved.
+
+Refer to the <<release-notes-6.3.0,6.3.0 Release Notes>> for a list bug
+fixes and other changes.
+
+[float]
+=== License management and {xpack} code
+
+Starting with version 6.3, all of the free {xpack} features, such
+as monitoring, tile maps, Grok Debugger, and Search Profiler,
```

Comment:
nit : we should either capitalize Monitoring , Tile maps just like Grok Debugger and Search Profiler - or make everything start with small letters?